src/quic/node_quic_util-inl.h

```diff
@@ -42,11 +42,12 @@ size_t QuicCID::Hash::operator()(const QuicCID& token) const {
   return hash;
 }
 
-bool QuicCID::Compare::operator()(
-    const QuicCID& lcid,
-    const QuicCID& rcid) const {
-  return lcid->datalen != rcid->datalen ?
-      false : memcmp(lcid->data, rcid->data, lcid->datalen) == 0;
+bool QuicCID::operator==(const QuicCID& other) const {
+  return memcmp(cid(), other.cid(), sizeof(ngtcp2_cid)) == 0;
```

Comment:
Does this mean that we assume a particular `datalen` now?

Reply:
the ngtcp2_cid struct uses a fixed size buffer now. The actual bits could be anything <= 20 bytes but the struct should be properly initialized.